modules/proxy/balancers/mod_lbmethod_bybusyness.c

```diff
@@ -25,18 +25,24 @@ module AP_MODULE_DECLARE_DATA lbmethod_bybusyness_module;
 static APR_OPTIONAL_FN_TYPE(proxy_balancer_get_best_worker)
                             *ap_proxy_balancer_get_best_worker_fn = NULL;
 
+
 static int is_best_bybusyness(proxy_worker *current, proxy_worker *prev_best, void *baton)
 {
     int *total_factor = (int *)baton;
 
     current->s->lbstatus += current->s->lbfactor;
     *total_factor += current->s->lbfactor;
+    apr_size_t current_busy = getbusy_count(current);
```

Comment:
To make the automated checks happy, you need to move the declarations in line 35 and 36 to the beginning of the block, at least before the first non-declaration line.